internal/controller/dscinitialization/dscinitialization_controller.go

```diff
@@ -483,7 +483,7 @@ func (r *DSCInitializationReconciler) newMonitoringCR(ctx context.Context, dsci
 		},
 	}
 
-	metricsEnabled := dsci.Spec.Monitoring.Metrics != nil && (dsci.Spec.Monitoring.Metrics.Storage != nil || dsci.Spec.Monitoring.Metrics.Resources != nil)
+	metricsEnabled := dsci.Spec.Monitoring.Metrics != nil && (dsci.Spec.Monitoring.Metrics.Storage != nil || dsci.Spec.Monitoring.Metrics.Resources != nil || len(dsci.Spec.Monitoring.Metrics.Exporters) > 0)
```

Comment:
Do we always expect Metrics.Exporters to be `> 0`? a user may enable metrics and just use the local prometheus/thanos querier, they may not export the metrics anywhere. This would enforce a user having to include an exporter right?

Reply:
yes it would